File: Codechef/html_tags.cpp
#include <bits/stdc++.h>
using namespace std;

int check(string n){
    int size = n.size();
    int flag = 0;
    if(n[0] != '<'){
        return 0;
    }
    else if(n[1] != '/'){
        return 0;
    }
    else if(size <= 3){
        return 0;
    }
    else if(n[size-1] != '>'){
        return 0;
    }
    else{
        for(int i =2 ;i < size-1;i++){
            if(!((n[i] >= 'a' && n[i] <= 'z') || (n[i] >= '0' && n[i] <= '9'))){
                return 0;
            }
        }
    }
    return 1;
}


int main() {
	int t;
	cin >> t;
	while(t--){
	    string n ;
	    cin >> n;
	    if(check(n)){
	        cout << "success" << endl;
	    }
	    else {
	        cout << "error" << endl;
	    }
	}
	return 0;
}

File: Codechef/HTML_Tags.java
import java.util.*;
class MyClass {
    public static void main(String args[]) {
      Scanner sc=new Scanner(System.in);
      int t=sc.nextInt();
      for(int y=1;y<=t;y++)
      {
          String c="";
          int flag=0;
          int counter=0;
          String s=sc.next();
          if(s.length()<4)
          {
               System.out.println("Error");
          }
          else{
          if(s.charAt(0)=='<' && s.charAt(1)=='/' && s.charAt(s.length()-1 )== '>')
          {
               for(int x=2;x<s.length()-1;x++)
          {
              char ch=s.charAt(x);
              if(((int)ch>=97 && (int)ch<=122) || ((int)ch>=48 && (int)ch<=57))
              {counter++;
              }else
              {flag=1;
              break;}
              
          }
          if(flag==0)
          {
              System.out.println("Success");
          }
          else
          { 
              System.out.println("Error");
          }}
           else
           { 
               System.out.println("Error");
         }}
          
      }
      
    }
}


// Input
// 5 
// </h1>  
// Clearly_Invalid  
// </singlabharat>  
// </5>  
// <//aA>  


// Output
// Success  
// Error  
// Success
// Success
// Error